BookingsApi/BookingsApi/Controllers/V1/JudiciaryParticipantsController.cs

```diff
@@ -0,0 +1,87 @@
+using BookingsApi.Contract.V1.Requests;
+using BookingsApi.Contract.V1.Responses;
+using BookingsApi.Mappings.Common;
+using BookingsApi.Mappings.V1;
+using BookingsApi.Validations.V1;
+using Microsoft.AspNetCore.Authorization;
+
+namespace BookingsApi.Controllers.V1
+{
+    [Produces("application/json")]
+    [Route("hearings")]
+    [ApiVersion("1.0")]
+    [ApiController]
+    public class JudiciaryParticipantsController : Controller
+    {
+        private readonly IQueryHandler _queryHandler;
+        private readonly ICommandHandler _commandHandler;
+        private readonly IHearingParticipantService _hearingParticipantService;
+        
+        public JudiciaryParticipantsController(
+            IQueryHandler queryHandler, 
+            ICommandHandler commandHandler,
+            IEventPublisher eventPublisher)
+        {
+            _queryHandler = queryHandler;
+            _commandHandler = commandHandler;
+            _hearingParticipantService = new HearingParticipantService(commandHandler, eventPublisher);
+        }
+        
+        /// <summary>
+        /// Add judiciary participants to a hearing
+        /// </summary>
+        /// <param name="hearingId">The id of the hearing</param>
+        /// <param name="request"></param>
+        [HttpPost("{hearingId}/joh")]
+        [OpenApiOperation("AddJudiciaryParticipantsToHearing")]
+        [ProducesResponseType(typeof(List<JudiciaryParticipantResponse>), (int)HttpStatusCode.OK)]
+        [ProducesResponseType(typeof(string), (int)HttpStatusCode.NotFound)]
+        [ProducesResponseType(typeof(ValidationProblemDetails), (int)HttpStatusCode.BadRequest)]
+        [AllowAnonymous]
+        [MapToApiVersion("1.0")]
+        public async Task<IActionResult> AddJudiciaryParticipantsToHearing(Guid hearingId, [FromBody] List<JudiciaryParticipantRequest> request)
+        {
+            var validation = await new AddJudiciaryParticipantsToHearingRequestValidation().ValidateRequestsAsync(request);
+            if (!validation.IsValid)
+            {
+                ModelState.AddFluentValidationErrors(validation.Errors);
+                return ValidationProblem(ModelState);
+            }
+
+            var participants = request
+                .Select(JudiciaryParticipantRequestToNewJudiciaryParticipantMapper.Map)
+                .ToList();
+
+            var command = new AddJudiciaryParticipantsToHearingCommand(hearingId, participants);
+            
+            try
+            {
+                await _commandHandler.Handle(command);
+            }
+            catch (HearingNotFoundException exception)
```

Comment:
Might be one for the future but if we had the not found exceptions extend from the same base class we could create a single catch block here

Reply:
that's a good idea. Probably one for future tech debt if that's ok with you, as all our not found exceptions work that way currently